Makefile

```diff
@@ -80,11 +86,7 @@ docker_ui:
 
 docker_mqtt:
 	# MQTT Docker build must be done from root dir because it copies .proto files
-ifeq ($(GOARCH), arm)
-	docker build --tag=mainflux/mqtt-arm -f mqtt/aedes/Dockerfile.arm .
-else
-	docker build --tag=mainflux/mqtt-amd64 -f mqtt/aedes/Dockerfile .
-endif
+	docker build --tag=mainflux/mqtt -f mqtt/aedes/Dockerfile.arm .
```

Comment:
Should we use `mqtt/aedes/Dockerfile` instead of `mqtt/aedes/Dockerfile.arm`?

Reply:
Damn, good catch!